packages/typescript/README.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Typescript
+---
+
+`@defimetachain/typescript` 
+
+For housing and keeping `tsconfig.json` DRY.
```

Comment:
What does it mean to keep the config `DRY`?

Reply:
**D**on't **R**epeat **Y**ourself